app/models/course.rb

```diff
@@ -316,6 +319,8 @@ def self.entry_requirement_options_without_nil_choice
   validates :name, :profpost_flag, :program_type, :qualification, :start_date, :study_mode, presence: true
   validates :age_range_in_years, presence: true, on: %i[new create publish], unless: :further_education_course?
   validates :level, presence: true, on: %i[new create publish]
+  validates :campaign_name, inclusion: { in: campaign_names.keys }
+  # TODO: validates :master_subject_id ?
```

Comment:
Remnant?

Reply:
Yes, see above, depends if we stick with `master_subject_id`